plutus-ledger-api/src/PlutusLedgerApi/Common/Eval.hs

```diff
@@ -119,17 +119,21 @@ newtype EvaluationContext = EvaluationContext
     deriving stock Generic
     deriving anyclass (NFData, NoThunks)
 
-{-|  Create an 'EvaluationContext' for a given plutus-builtins' version.
+{-|  Create an 'EvaluationContext' for a given builtin semantics variant.
 
```

Comment:
Is that right?  I presume it's not referring to the ledger language version, although this is in fact what gets called in the V1/V2/V2 `mkEvaluationContext` functions.

Reply:
Yes, I think it is right. It's getting called with the corresponding `BuiltinSemanticsVariant`. Possibly we should just use this generically, and instead have for each language a clearly associated `BuiltinSemanticsVariant` or something.